README.md

```diff
@@ -36,10 +36,10 @@ If nothing helps and you are stuck, these are places you can go ask for guidance
 Restart the DFX network with:
 
 ```
-dfx start --clean
+dfx start --clean --background
```

Comment:
I don't see why running it in the background is necessary. I personally like to be able to see the output.

Reply:
You'll see the output. You just don't need to open another shell.

Reply:
I can remove it, if you like.

Reply:
yeah, I'd rather remove it, thanks! The important thing about this troubleshooting solution is the `--clean` part, so I'd rather keep the solution simple and clear.

Reply:
It's gone!